web/source/kmwosk.ts

```diff
@@ -2863,8 +2863,8 @@ if(!window['keyman']['initialized']) {
       for(k = 0; k < t.childNodes.length; k++)
       {
         if(t.childNodes[k].firstChild.className.indexOf('key-hidden') >= 0) continue;
-        x1 = t.childNodes[k].firstChild.offsetLeft;
-        x2 = x1 + t.childNodes[k].firstChild.offsetWidth;
+        x1 = t.childNodes[k].offsetLeft;
```

Comment:
Note: WKWebView treated this differently to all other browsers in testing. `t.childNodes[k].firstChild.offsetLeft` was always calculated relative to its parent element (i.e. `offsetParent != null`). However, for all other browsers `offsetParent == null` was `true`. We should consider moving to using `getBoundingClientRect()` everywhere in the future for better compatibility?